modules/indexExchangeBidAdapter.js

```diff
@@ -269,11 +269,24 @@ var cygnus_index_start = function () {
     this.siteID = siteID;
     this.impressions = [];
     this._parseFnName = undefined;
+
+    // Get page URL
+    this.sitePage = undefined;
+    try {
+      this.sitePage = utils.getTopWindowUrl();
```

Comment:
Hi @mkendall07. Thank you for the follow-up. We will fix this when #1530 is merged in.